packages/dcos-integration-test/extra/test_remove_dockercfg_hook.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+
+import pytest
+
+
+@pytest.fixture(scope='module')
+def auth_cluster(cluster):
```

Comment:
Or maybe it only works on vagrant since the test is run from a master and the agent is trying to fetch from localhost.